src/store/CrosshairStore.js

```diff
@@ -19,122 +20,57 @@ class Tooltip extends CIQ.Marker {
             chartContainer: true,
             label: 'tooltip',
         });
+
         this.stx = stx;
         this.setRows = setRows;
         this.show = show;
         this.hide = hide;
-        this.lastBar = { };
+        this.lastBar = {};
 
         this.className = 'CIQ.Marker.Tooltip';
         this.showChange = false;
         this.showSeries = true;
         this.showStudies = true;
 
-        CIQ.ChartEngine.prototype.append('undisplayCrosshairs', () => {
```

Comment:
undisplayCrosshairs injection is not required since renderFunction already checks whether crosshairs are rendered